src/platform/graphics/graphics-device.js

```diff
@@ -39,6 +39,13 @@ class GraphicsDevice extends EventHandler {
      */
     isWebGPU = false;
 
+    /**
+     * The type of graphics device that the system could initialize. Can be one of pc.DEVICETYPE_WEBGL1, pc.DEVICETYPE_WEBGL2 or pc.DEVICETYPE_WEBGPU.
+     *
+     * @type {import('./constants.js').DEVICETYPE_WEBGL1 | import('./constants.js').DEVICETYPE_WEBGL2 | import('./constants.js').DEVICETYPE_WEBGPU}
+     */
+    activeDeviceType;
```

Comment:
change it to deviceType please, 'active' suggests it can change, which it cannot. And the description:
`A type of the device. Can be one of ...`